assets/src/edit-story/components/panels/pageStyle.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Color, Row } from '../form';
+import { useStory } from '../../app';
+import createSolid from '../../utils/createSolid';
+import { SimplePanel } from './panel';
+import getColorPickerActions from './utils/getColorPickerActions';
+
+function PageStylePanel() {
+  const {
+    state: { currentPage },
+    actions: { updateCurrentPageProperties },
+  } = useStory();
+  const backgroundColor =
+    currentPage.backgroundColor || createSolid(255, 255, 255, 1);
```

Comment:
Nit: `alpha=1` is default and not necessary. The default is applied when generating the pattern for a solid color with missing alpha channel (to make the saved object smaller).

Reply:
Good point. This could actually be removed perhaps if we're adding the default bg to the newly created pages + migration should take care of the previously existing pages.